app/MenuTitle.java

```diff
@@ -36,11 +38,12 @@ public static final void displayStoreName(String name) {
         MenuTitle.displayBorder();
     }
 
-    public static final void displayOptions(String[] optionsList) {
+    public static final void displayOptions(ArrayList<String> optionsList) {
```

Comment:
you can rename `optionsList` to `options` kay implied nga list sya from its type